metadata-models/src/main/pegasus/com/linkedin/metadata/query/GroupingCriterion.pdl

```diff
@@ -0,0 +1,20 @@
+namespace com.linkedin.metadata.query
+
+/**
+*
+**/
+
+record GroupingCriterion {
+
+    /**
+    * The type of the entity to be grouped.
+    * e.g. schemaField
+    */
+    rawEntityType: string
```

Comment:
one optimization on model: 

make this optional, which means "applies to all entity types"